modules/round/src/main/Round.scala

```diff
@@ -200,15 +196,12 @@ private[round] final class Round(
     }
 
     case DeployPost => handle { game =>
-      game.clock.filter(_ => game.playable) ?? { clock =>
-        val freeSeconds = 15
-        val freeCentis = Centis.ofSeconds(freeSeconds)
-        val newClock = clock.giveTime(Color.White, freeCentis).giveTime(Color.Black, freeCentis)
-        val progress = (game withClock newClock) + Event.Clock(newClock)
+      game.playable ?? {
+        val freeTime = 15.seconds
         messenger.system(game, (_.untranslated("Lichess has been updated")))
         messenger.system(game, (_.untranslated("Sorry for the inconvenience!")))
-        Color.all.foreach { c =>
-          messenger.system(game, (_.untranslated(s"$c + $freeSeconds seconds")))
+        val progress = giveMoretime(game, Color.White, freeTime) flatMap {
```

Comment:
I was having trouble telling if this will order the clock events correctly.  We need the second movetime event to come after the first.